public/styles/css/main.css

```diff
@@ -2,76 +2,126 @@
 *            Helpers           *
 ********************************/
 .italic {
-  font-style: italic;
-  color: #202020; }
+  font-style: italic; }
 
```

Comment:
Hey, there should be a variable for this instead of writing out the hex code for the color.

Reply:
Thanks Cynthia! (but that was actually removed 🤓)